### ISSUE TITLE
Hooks looks broken.

### ISSUE THREAD

**Issue by buglocker** [justdanpo/VivaldiHooks#78]
Hi. 
After the latest upgrade to version 4.3.2439.39 hooks wont work. Is that real or my system have a problem ?
Thanks.

**Comment by stardepp** [comment]
No, this affects all users of Vivaldi from version 4.3. See here in the forum: https://forum.vivaldi.net/topic/11298/vivaldihooks-more-useful-mods?page=1
